pkgs/applications/networking/eddie-ui/default.nix

```diff
@@ -0,0 +1,92 @@
+{ stdenv,
+  fetchFromGitHub,
+  clang,
+  cmake,
+  mono,
+  dotnet-sdk,
+  which,
+  glibc,
+  pkgconfig,
+  gtk2,
+  libappindicator-gtk2,
+  lzma,
+  libselinux,
+  libsepol,
+  fribidi,
+  libthai,
+  libdatrie,
+  xorg,
+  openvpn,
+  stunnel
+  }:
+
+stdenv.mkDerivation rec {
+  pname = "eddie-ui";
+  version = "2.18.9";
+  arch = if stdenv.system == "i686-linux"
+    then "x86"
+    else "x64";
+  src = fetchFromGitHub {
+    owner = "AirVPN";
+    repo = "Eddie";
+    rev = "2.18.9";
+    sha256 = "1nvnvy6lskydfrlsrbyk065hl6zxdyivhrynydny1kirz3qmrw61";
+  };
+  sourceRoot= "/build/source/src/UI.GTK.Linux.Tray";
```

Comment:
```suggestion

```

IIRC this is not required here.

Reply:
If I remove this line, I need to add 
`dontUseCmakeConfigure=true;`
Otherwise I get the error:
`CMake Error: The source directory "/build/source" does not appear to contain CMakeLists.txt.`
Also, when I skip configure phase with the option above I have an error while building:
```
/build/source/src/UI.GTK.Linux.Tray/src/main.cpp:22:10: fatal error: libappindicator/app-indicator.h: No such file or directory
   22 | #include <libappindicator/app-indicator.h>
```
Looks like in this case I need to provide some cmake flags manually.